pynestml/meta_model/ast_for_stmt.py

```diff
@@ -69,7 +69,7 @@ def clone(self):
         """
         variable_dup = None
         if self.variable:
-            variable_dup = self.variable.clone()
+            variable_dup = str(self.variable)
```

Comment:
Can we just assign ``variable_dup = self.variable``? Strings are immutable in Python. Same goes for the float and int below.

Reply:
Sure!